app/instance-initializers/ember-i18n.js

```diff
@@ -13,7 +14,14 @@ export default {
     }
     instance.container.lookup('service:i18n').set('locale', defaultLocale);
 
-    Ember.HTMLBars._registerHelper('t', helper);
+    if (legacyHelper != null) {
+      Ember.HTMLBars._registerHelper('t', legacyHelper);
+    }
+
+    if (Helper != null) {
+      instance.registry.register('helper:t', Helper);
```

Comment:
Ah. I was basing this off of https://github.com/emberjs/ember.js/pull/11278/files#diff-5d4704d516d9bedb9605e499dc0a88d8R122

I guess once dashless-autolookup lands I'll want a single `app/helpers/t.js` that exports either `legacyHelper` or `Helper`, depending on the presence of `Ember.Helper`. That'll be easy :)